hydroshare/local_settings.py

```diff
@@ -179,3 +179,5 @@
 #DEFAULT_SUPPORT_EMAIL=''
 
 HYDROSHARE_SHARED_TEMP = '/shared_tmp'
+
+TIME_ZONE = "Etc/UTC"
```

Comment:
@alexlemann - the Docker container defaults to UTC as it's timezone, so keeping this as UTC is fine.